stdlib/2/__builtin__.pyi

```diff
@@ -1370,14 +1370,16 @@ else:
     # This is only available after from __future__ import print_function.
     def print(*values: object, sep: Optional[Text] = ..., end: Optional[Text] = ..., file: Optional[_Writer] = ...) -> None: ...
 
-@overload
-def pow(__x: int, __y: int) -> Any: ...  # The return type can be int or float, depending on y
-@overload
-def pow(__x: int, __y: int, __z: int) -> Any: ...
-@overload
-def pow(__x: float, __y: float) -> float: ...
-@overload
-def pow(__x: float, __y: float, __z: float) -> float: ...
+if sys.version_info >= (3, 8):
+    @overload
+    def pow(base: int, exp: int, mod: Optional[int] = ...) -> int: ...
```

Comment:
Needs to return `Any` per previous comment. E.g. `pow(3, -1)` returns a float. It makes sense to use another overload for the case where `mod` is an `int`, since then only `int` can be returned. (Affects both branches.)

Reply:
Oh of course. Sorry about that, total mental lapse